app/models/photosynthesis_measurement.rb

```diff
@@ -1,4 +1,42 @@
+require 'csv'
+
 class PhotosynthesisMeasurement < ActiveRecord::Base
   include BatchImport
+  
+  belongs_to :leaf_part
+
+  validates :leaf_part_id, presence: true
+
+  validates :filename, presence: true
```

Comment:
Aligning the second parameter to make it more readable is nice for some people. But I'm quite dyslexic and I find wider whitespace makes me "get lost".